Source/Editor/DualityEditor/Controls/AutoExpandTreeView.cs

```diff
@@ -0,0 +1,68 @@
+using Aga.Controls.Tree;
+using System;
+using System.Drawing;
+using System.Windows.Forms;
+
+namespace Duality.Editor.Controls
+{
+	public class AutoExpandTreeView : TreeViewAdv
+	{
+		public int DragOverExpandDelay
```

Comment:
We generally don't do auto-properties in Duality source. Please replace with a regular field-property combo.